projects/addon-editor/components/editor/editor.component.ts

```diff
@@ -161,6 +161,14 @@ export class TuiEditorComponent
         this.editor?.unsetLink();
     }
 
+    focus(): void {
+        (
+            this.el?.nativeElement.querySelector(
```

Comment:
Is there a way to get it without querying the DOM? Also it should be `nativeFocusableElement`. I know we haven't solved it not being available for some time for autofocus, but that's a separate issue. Let's fix `nativeFocusableElement` getter to target the correct element.

Reply:
not affected

<img width="1248" alt="image" src="https://github.com/Tinkoff/taiga-ui/assets/12021443/60057046-0c14-4975-9e26-04d829db05c6">
<img width="555" alt="image" src="https://github.com/Tinkoff/taiga-ui/assets/12021443/1d587e70-0fd4-4b3d-ad08-2683259133c5">

Reply:
I meant encapsulate getting the **actual focusable element** inside `nativeFocusableElement` getter, that is **what it is for**. The issue of it not being immediately available is a separate one which we will resolve later, it is irrelevant to focusing on click since at the time you click it should already be there.